.rubocop_todo.yml

```diff
@@ -0,0 +1,745 @@
+# Configuration parameters: AllowSafeAssignment.
+Lint/AssignmentInCondition:
+  Exclude:
+    - 'benchmarks/index_v_take_while.rb'
+    - 'spec/rspec/core/metadata_spec.rb'
+
+# Offense count: 2
+Lint/BooleanSymbol:
+  Exclude:
+    - 'spec/rspec/core/example_group_spec.rb'
+
+# Offense count: 2
+Lint/HandleExceptions:
+  Exclude:
+    - 'lib/rspec/core/example.rb'
+    - 'lib/rspec/core/mocking_adapters/mocha.rb'
+    - 'lib/rspec/core/runner.rb'
+    - 'lib/rspec/core/test_unit_assertions_adapter.rb'
+    - 'script/rspec_with_simplecov'
+    - 'spec/rspec/core/example_spec.rb'
+
+# Offense count: 2
+Lint/InterpolationCheck:
+  Exclude:
+    - 'spec/rspec/core/bisect/runner_spec.rb'
+    - 'spec/rspec/core/formatters/exception_presenter_spec.rb'
+
+# Offense count: 1
+Lint/ShadowingOuterLocalVariable:
+  Exclude:
+    - 'benchmarks/hash_functions.rb'
+
+# Offense count: 22
+# Cop supports --auto-correct.
+Lint/UnneededSplatExpansion:
+  Exclude:
+    - 'spec/rspec/core/configuration_options_spec.rb'
+    - 'spec/rspec/core/drb_spec.rb'
+
+# Offense count: 24
+# Cop supports --auto-correct.
+# Configuration parameters: IgnoreEmptyBlocks, AllowUnusedKeywordArguments.
+Lint/UnusedBlockArgument:
+  Exclude:
+    - 'features/support/env.rb'
+    - 'features/support/jruby.rb'
+    - 'features/support/rubinius.rb'
+    - 'spec/rspec/core/example_group_spec.rb'
+    - 'spec/rspec/core/formatters/exception_presenter_spec.rb'
+    - 'spec/rspec/core/formatters/json_formatter_spec.rb'
+    - 'spec/rspec/core/invocations_spec.rb'
+    - 'spec/rspec/core/metadata_filter_spec.rb'
+    - 'spec/rspec/core/metadata_spec.rb'
+    - 'spec/rspec/core/notifications_spec.rb'
+    - 'spec/rspec/core/rake_task_spec.rb'
+    - 'spec/rspec/core/reporter_spec.rb'
+    - 'spec/support/formatter_support.rb'
+
+# Offense count: 1
+# Cop supports --auto-correct.
+# Configuration parameters: AllowUnusedKeywordArguments, IgnoreEmptyMethods.
+Lint/UnusedMethodArgument:
+  Exclude:
+    - 'benchmarks/capture_block_vs_yield.rb'
+
+# Offense count: 1
+Lint/UselessAssignment:
+  Exclude:
+    - 'benchmarks/hash_functions.rb'
+
+# Offense count: 2
+Lint/UselessComparison:
+  Exclude:
+    - 'spec/rspec/core/example_spec.rb'
+
+# Offense count: 2
+Lint/Void:
+  Exclude:
+    - 'spec/rspec/core/example_group_spec.rb'
+
+# Offense count: 1
+Metrics/AbcSize:
+  Max: 283
+
+# Offense count: 32
+# Configuration parameters: CountComments, ExcludedMethods.
+Metrics/BlockLength:
+  Max: 2222
+
+# Offense count: 68
+# Configuration parameters: AllowHeredoc, AllowURI, URISchemes, IgnoreCopDirectives, IgnoredPatterns.
+# URISchemes: http, https
+Metrics/LineLength:
+  Max: 172
+
+# Offense count: 1
+# Configuration parameters: CountComments.
+Metrics/MethodLength:
+  Max: 134
+
+# Offense count: 30
+# Configuration parameters: CountComments.
+Metrics/ModuleLength:
+  Max: 2200
+
+# Offense count: 1
+# Configuration parameters: CountKeywordArgs.
+Metrics/ParameterLists:
+  Max: 6
```

Comment:
The second was this. There is one other place in the codebase that uses 6 method parameters. Since the todo detected this and raised the overall `Max`, the inline whitelisting became unneeded. Regenerating the todo reveals that the deletion I made raised the `Offense count` to `2`.  

Unfortunately, `Metrics` cops do not list exclusions the way all the other cop types do. So this is undesirable. I'll see if I can find the other offense and inline disable it.